bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableClient.scala

```diff
@@ -101,8 +102,9 @@ trait CachedBackend extends VoTableBackend {
       @tailrec
       def go(pos: Int):Option[(Int, QueryResult)] =
         a.lift(pos) match {
-          case None    => None
-          case Some(x) => if (x.k.query.isSuperSetOf(k.query)) Some((pos, x.v)) else go(pos + 1)
+          case None                                                            => None
+          case Some(CacheEntry(SearchKey(query:ConeSearchCatalogQuery, r), v)) => if (query.isSuperSetOf(k.query)) Some((pos, v)) else go(pos + 1)
+          case Some(_)                                                         => None // Not caching named queries so far
```

Comment:
I thought about adding caching to `NamedQuery` right away but then decided against it so it would have used `Some(CacheEntry(SearchKey(query:NamedQuery, r), v))`
So, you are right, it can be replaced by `_`